src/prognostic_equations/vertical_diffusion_boundary_layer.jl

```diff
@@ -96,23 +96,22 @@ function vertical_diffusion_boundary_layer_tendency!(
 
     ᶜρχₜ_diffusion = p.scratch.ᶜtemp_scalar
     ᶜK_h_scaled = p.scratch.ᶜtemp_scalar_2
-    for (ᶜρχₜ, ᶜχ, χ_name) in matching_subfields(Yₜ.c, ᶜspecific)
-        χ_name == :e_tot && continue
-        if χ_name in (:q_rai, :q_sno, :n_rai)
+    ᶜdivᵥ_ρχ = Operators.DivergenceF2C(
+        top = Operators.SetValue(C3(0)),
+        bottom = Operators.SetValue(C3(0)),
+    )
+    foreach_scalar(Yₜ, Y) do ᶜρχₜ, ᶜρχ, is_ρq_tot
+        if ρχ_name in (:ρq_rai, :ρq_sno, :ρn_rai)
```

Comment:
@tapios, this is the only remaining unresolved issue (`ρχ_name` is not emitted by `foreach_scalar`), as far as I can tell.

Do you want me to add `ρχ_name` to the args and update all of the calls accordingly?

Reply:
Yes, why not.